desec.py

```diff
@@ -1427,6 +1484,47 @@ def tlsa_record(
     return f"{int(usage)} {int(selector)} {int(match_type)} {hex_data}"
 
 
+class CliClientFormatter(Formatter):
+    """Pretty prints requests and response logs for CLI usage."""
+
+    def format(self, record):
+        """Format a log record for cli usage.
+
+        Args:
+            record: Log record to format.
+
+        Returns:
+            Formatted log record as string.
+        """
+        message = record.getMessage()
+        if getattr(record, "params", None):
+            message += "\nParams:"
+            for k, v in record.params.items():
+                message += f"{k}: {v}"
+        if getattr(record, "body", None):
+            message += "\nBody:\n"
+            message += json.dumps(record.body, indent=2)
+        if getattr(record, "response_body", None):
+            message += "\n"
+            message += json.dumps(record.response_body, indent=2)
+            message += "\n"
+        return message
+
+
+def configure_cli_logging(level):
+    """Set up logging configration when using the module as a command-line interface.
+
+    Args:
+        level: Logging level to set for desec.client logger.
+    """
+    http_handler = StreamHandler(stream=sys.stdout)
+    http_formatter = CliClientFormatter()
+    http_handler.setFormatter(http_formatter)
+    http_logger = getLogger("desec.client")
```

Comment:
Any particular reason for naming the logger instead of just using `__name__`?

Reply:
See comment above